<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>进程调度模拟器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="dataTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>451</width>
      <height>141</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="resultTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>241</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>任务序列</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>执行结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteTask">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>451</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>进程名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="taskName"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>创建时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="createdTime"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>服务时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="serveTime"/>
     </item>
     <item>
      <widget class="QPushButton" name="addTask">
       <property name="text">
        <string>添加任务</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="ScheduleView" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>451</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>调度图</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="slice">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>290</y>
      <width>48</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>476</x>
      <y>242</y>
      <width>52</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>调度方式</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="FCFS">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>260</y>
      <width>58</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FCFS</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="SJF">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>311</y>
      <width>48</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SJF</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="HRN">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HRN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="run">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>68</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>运行</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="RR">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>290</y>
      <width>42</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>RR</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>431</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ScheduleView</class>
   <extends>QWidget</extends>
   <header>scheduleview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
